install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,67 +11,68 @@
 toc::[]
 
 == Overview
-You can provision your OpenShift cluster with
-link:../../architecture/additional_concepts/storage.html[persistent storage] using
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS].
-Some familiarity with Kubernetes and NFS is assumed.
-
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
-
-For a detailed example, see the guide for
-https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
-MySQL using NFS].
 
-[IMPORTANT]
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage]
+using NFS. Some familiarity with Openshift,
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent
+volume claims (PVCs)], and
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS]
+is beneficial.
+
+The Openshift persistent volume (PV) framework allows administrators to provision a cluster
+with persistent storage and gives developers the ability to request those resources without
+having specific knowledge of the underlying infrastructure. Persistent volumes are not bound
+to a single project; they can be shared across the entire OpenShift cluster. Persistent volume
```

Comment:
I would add a qualification to this sentence; as written it sounds to me like you can share PVs all over town when the reality is that claims cannot be shared across namespaces.  It might be better to introduce claims first, and say that claims are required to use a PV, and they are namespaced, but the volumes themselves can be claimed from any project.